lib/chat_api/slack/notifications.ex

```diff
@@ -143,7 +143,9 @@ defmodule ChatApi.Slack.Notifications do
       message = %{
         "text" => format_slack_message_text(message),
         "channel" => thread.slack_channel,
-        "thread_ts" => thread.slack_thread_ts
+        "thread_ts" => thread.slack_thread_ts,
+        "username" => format_user_name(user),
```

Comment:
(still need to test how this is handled when the required scopes are missing... don't merge until that's verified)

Reply:
tested, seems to just ignore these fields when scope is missing 👍